doc/release-notes/release-4.sgml

```diff
@@ -176,10 +176,14 @@ Most user-facing changes are reflected in squid.conf (see below).
 
 
 <sect1>Initial GnuTLS support
-<p>If all you need is a proxy that connects over TLS/SSL to a <em>cache_peer</em>
-   or accepts <em>https://</em> URLs over clear-text and performs the necessary
-   upstream TLS connections. Then you now have the choice to build Squid with
-   GnuTLS instead of OpenSSL.
+<p>Squid can now be built to use GnuTLS in place of OpenSSL for the core
+   features of receiving TLS connections from clients, and making TLS
+   connection to servers. The GnuTLS support is still very much experimental
```

Comment:
s/connection/connections/ for consistency with the previous "receiving" phrasing

Reply:
done